### ISSUE TITLE
Can't change editor shortcuts for next/previous script

### ISSUE THREAD

**Issue by haimat** [godotengine/godot#43785]
**Godot version:**
3.2.3 stable

**OS/device including version:**
Windows 10
OpenGL ES 3.0 Renderer: GeForce GTX 1080/PCIe/SSE2

**Issue description:**
I opened the "Shortcuts" tab in the editor settings and changed the "Next script" and "Previous script" shortcuts under the "Script Editor" section to <kbd>Ctrl</kbd>+<kbd>PageUp</kbd> / <kbd>Ctrl</kbd>+<kbd>PageDown</kbd>, respectively. Originally they are <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>,</kbd> and <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>.</kbd>. However, after settings these shortcuts and pressing them in the text edtior nothing happens, the current/selected script does not change.

**Comment by Calinou** [comment]
Maybe this is because <kbd>Ctrl</kbd> + <kbd>PageUp</kbd> and <kbd>Ctrl + PageDown</kbd> is already used in the script editor itself? Therefore, the script editor intercepts the shortcut.

**Comment by haimat** [comment]
> Maybe this is because Ctrl + PageUp and Ctrl + PageDown is already used in the script editor itself? Therefore, the script editor intercepts the shortcut and considers the input event as handled.

I thought about so too, but when searching for another shortcut using <kbd>Ctrl</kbd>+<kbd>PageUp</kbd> or <kbd>Ctrl</kbd>+<kbd>PageDown</kbd>, I couldn't find any. So I don't see that anything else is using these shortcuts. Or are they built-in and therefore not an option to change in the editor settings?

**Comment by EricEzaM** [comment]
I think it's used internally in text edit maybe? So it won't be listed in the editor shortcuts section. #43663 addresses this issue.

**Comment by Rprabudeva** [comment]
I would like to contribute

**Comment by Calinou** [comment]
@Rprabudeva This is not a trivial issue to solve. I'd look at issues labeled [junior job](https://github.com/godotengine/godot/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22junior+job%22) instead. Make sure to pick an issue which is not being addressed by a pull request already (check the column on the right for "pull request" icons, there should be none.)=

**Comment by haimat** [comment]
So do I understand correctly that for now I cannot use <kbd>Ctrl</kbd>+<kbd>PageUp</kbd> or <kbd>Ctrl</kbd>+<kbd>PageDown</kbd> for my custom editor shortcuts?

**Comment by Calinou** [comment]
@haimat Indeed, I don't think that'll work until https://github.com/godotengine/godot/pull/43663 is merged.

**Comment by dark-penguin** [comment]
I've encountered this before: https://github.com/godotengine/godot/issues/39780
I ended up with Ctrl+Alt+Up/Down to switch scripts.
